specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json

```diff
@@ -224,14 +224,55 @@
         "tags": [
           "Library"
         ],
-        "operationId": "Library_CreateOrAppend",
+        "operationId": "Library_Create",
         "x-ms-examples": {
           "Create Library": {
-            "$ref": "./examples/Library_CreateOrAppend.json"
+            "$ref": "./examples/Library_Create.json"
           }
         },
-        "description": "Creates a library with the library name. Use query param 'comp=appendblock' to append the data to the library resource created using the create operation.",
+        "description": "Creates a library with the library name.",
         "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LibraryNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
An operation can't return both 200 and 202 - if 202 is ever possible, then 202 must always be the response. This rule exists so that customer code knows to properly deal with the response (especailly for SDKs) and also because the response payload tends to be different between 200 and 202 and an SDK can't different kinds of data types from a single method.

Reply:
200 response is enforced by the checks. One of the checks was failing because it required 200 for a PUT api if it is a long-running operation.